challenges/objects-arrays.js

```diff
@@ -6,120 +6,232 @@
-{"id":5,"first_name":"Chick","university":"Sultan Salahuddin Abdul Aziz Shah Polytechnic","email":"camorts4@google.com.au"},
-{"id":6,"first_name":"Jakob","university":"Fachhochschule Rosenheim, Hochschule für Technik und Wirtschaft","email":"jharken5@spiegel.de"},
-{"id":7,"first_name":"Robbi","university":"Salem University","email":"rbrister6@redcross.org"},
-{"id":8,"first_name":"Colline","university":"Coastal Carolina University","email":"cbrosh7@alibaba.com"},
-{"id":9,"first_name":"Michail","university":"Universidad Católica de Ávila","email":"mrome8@shinystat.com"},
-{"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
+const graduates = [
+  {
+    id: 1,
+    first_name: "Cynde",
+    university: "Missouri Southern State College",
+    email: "ctorry0@macromedia.com"
+  },
+  {
+    id: 2,
+    first_name: "Saundra",
+    university: "The School of the Art Institute of Chicago",
+    email: "swhal1@state.gov"
+  },
+  {
+    id: 3,
+    first_name: "Lambert",
+    university: "Marian College",
+    email: "lparham2@techcrunch.com"
+  },
+  {
+    id: 4,
+    first_name: "Modestine",
+    university: "International Medical & Technological University",
+    email: "mdolder3@symantec.com"
+  },
+  {
+    id: 5,
+    first_name: "Chick",
+    university: "Sultan Salahuddin Abdul Aziz Shah Polytechnic",
+    email: "camorts4@google.com.au"
+  },
+  {
+    id: 6,
+    first_name: "Jakob",
+    university:
+      "Fachhochschule Rosenheim, Hochschule für Technik und Wirtschaft",
+    email: "jharken5@spiegel.de"
+  },
+  {
+    id: 7,
+    first_name: "Robbi",
+    university: "Salem University",
+    email: "rbrister6@redcross.org"
+  },
+  {
+    id: 8,
+    first_name: "Colline",
+    university: "Coastal Carolina University",
+    email: "cbrosh7@alibaba.com"
+  },
+  {
+    id: 9,
+    first_name: "Michail",
+    university: "Universidad Católica de Ávila",
+    email: "mrome8@shinystat.com"
+  },
+  {
+    id: 10,
+    first_name: "Hube",
+    university: "Universitat Rovira I Virgili Tarragona",
+    email: "hlethbrig9@foxnews.com"
+  }
+];
 
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
-
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+const universities = [graduates]
+console.log(graduates)
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
-
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
-
 Log the result of your new array. */
-const contactInfo = [];
-console.log(contactInfo);
+const contactInfo = graduates.map((state)) {  
```

Comment:
You need to remember to check your work! No one is cool enough to code blind. Keep your quokka running or the dev tools with the index.html/live server. 

This is a syntax error which is killing your whole script. One `)`, killing it all. 

```suggestion
const contactInfo = graduates.map((state) {  
```